prutsen/prutsen.pde

```diff
@@ -11,27 +10,34 @@ float minGain = -100;
 int BPM = 0;                                                                     // Creating BPM var 0 as starting point
```

Comment:
je kunt nog eens kijken hoe het eruit ziet als je hier weer `float` van maakt. kan zijn dat je op een paar plaatsen soortgelijke veranderingen moet maken maar dat is better practice